test/BuildingRegistry.Tests/BackOffice/Api/WhenCorrectingBuildingUnitPosition/GivenBuilding.cs

```diff
@@ -0,0 +1,136 @@
+namespace BuildingRegistry.Tests.BackOffice.Api.WhenCorrectingBuildingUnitPosition
+{
+    using System;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Be.Vlaanderen.Basisregisters.GrAr.Edit.Contracts;
+    using BuildingRegistry.Api.BackOffice.Abstractions;
+    using BuildingRegistry.Api.BackOffice.Abstractions.BuildingUnit.Requests;
+    using BuildingRegistry.Api.BackOffice.Abstractions.BuildingUnit.Validators;
+    using BuildingRegistry.Api.BackOffice.Building;
+    using BuildingRegistry.Api.BackOffice.BuildingUnit;
+    using Building;
+    using FluentAssertions;
+    using FluentValidation;
+    using Moq;
+    using SqlStreamStore;
+    using Xunit;
+    using Xunit.Abstractions;
+    using Be.Vlaanderen.Basisregisters.Sqs.Responses;
+
+    public class GivenBuilding : BackOfficeApiTest
+    {
+        private readonly BuildingUnitController _controller;
+        private readonly Mock<IStreamStore> _streamStoreMock;
+
+        public GivenBuilding(ITestOutputHelper testOutputHelper) : base(testOutputHelper)
+        {
+            var buildings = new Mock<IBuildings>();
+            var backOfficeContext = new Mock<BackOfficeContext>();
+            _controller = CreateBuildingUnitControllerWithUser<BuildingUnitController>();
+            _streamStoreMock = new Mock<IStreamStore>();
+        }
+
+        [Fact]
+        public async Task ThenAcceptedResponseIsExpected()
```

Comment:
Duplicate test, it also exists in test/BuildingRegistry.Tests/BackOffice/Api/WhenCorrectingBuildingUnitPosition/GivenBuildingUnitPlanned.cs